src/app/components/media/MediaAnalysis.js

```diff
@@ -93,8 +101,9 @@ const MediaAnalysis = ({ projectMedia }) => {
     setEditing(false);
     if (value !== getValue(field) && canEdit) {
       setSaving(true);
-      const fields = {};
+      let fields = {};
       fields[field] = value;
+      fields = setInitialDatePublished(fields, field);
```

Comment:
Code looks great, but it doesn't adress the source of the confusion faced by users, developers and product team together :sweat_smile: